server/workers/github.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+if (!module.parent) {
+  // Set default node environment to development
+  process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
+  var mongoose = require('mongoose');
+  var config = require('../config/environment');
+  var args = process.argv.slice(2);
+
+  // Connect to database
+  var db = mongoose.connect(config.mongo.uri, config.mongo.options);
+}
+var Commit = require('../api/commit/commit.model');
+var Project = require('../api/project/project.model');
+
+var octo = require('octonode');
+/*
+ @TODO: IMPORT TOKEN FROM AN UNTRACKED/PRIVATE CONFIG FILE
+ */
+
+var token = config.GITHUB_WORKER_TOKEN;
+var octoclient = octo.client(token);
+
+Project.find().exec().then(
+  function (projects) {
```

Comment:
It would be better if this was modular and could be run on only a specific set of projects